pom.xml

```diff
@@ -51,6 +49,12 @@
     </developer>
   </developers>
 
+  <modules>
+    <module>sonar-maven-plugin</module>
+    <module>its</module>
+    <module>property-dump-plugin</module>
```

Comment:
If it does not have any impact on the build, let's have them listed in alphabetical order for good measure